ci/coverage-report-generator/Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'simplecov', '~> 0.17.1'
```

Comment:
Are we still using [this](https://github.com/cyberark/conjur/blob/master/Gemfile#L54) thing? If not, it probably should be deleted but I'm guessing that we may want to consider just a separate deployment type and having simplecov as part of conjur gemset maybe? WDYT?

Reply:
I'm not a bundler expert, but it looks like for that to work, every gem would need to be in a group so they could be excluded, probably easier to just keep the separate `Gemfile`. Unless theres a super strong reason for conjur and this random script to have to use the same verison of `simplecov`?

Reply:
Yeah fair enough

Reply:
Then we should remove the one in main gemfile then